versioned_docs/version-2.x/advanced/payments/stripe.mdx

```diff
@@ -182,3 +182,31 @@ See the tests for an example (while a detailed documentation is being written):
 
 - https://gitlab.blackswift.cloud/front-commerce/front-commerce/blob/3c550dfa0142dde7da3761011379118612841de7/src/server/modules/payment-stripe/__tests__/loader.js#L77
 - https://gitlab.blackswift.cloud/front-commerce/front-commerce/blob/3c550dfa0142dde7da3761011379118612841de7/src/server/modules/payment-stripe/__tests__/loader.js#L147
+
+### Advanced: network retries
+
+<SinceVersion tag="2.27" />
+
+Optionally the environment variable `FRONT_COMMERCE_STRIPE_MAX_NETWORK_RETRY`
+can be set to an integer [to configure the Stripe client `maxNetworkRetries`
+option](https://github.com/stripe/stripe-node#network-retries).
```

Comment:
**nitpick:** upon reading, it seems that without defining the variable there is no retries.

**suggestion:** mention the default value
```suggestion
Optionally the environment variable `FRONT_COMMERCE_STRIPE_MAX_NETWORK_RETRY`
can be set to an integer [to configure the Stripe client `maxNetworkRetries`
option](https://github.com/stripe/stripe-node#network-retries). It defaults to `3`.
```